ros2bag/ros2bag/verb/play.py

```diff
@@ -27,6 +27,9 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
         parser.add_argument(
             '-s', '--storage', default='sqlite3',
             help='storage identifier to be used, defaults to "sqlite3"')
+        parser.add_argument(
+            '-c', '--clock', default='past',
```

Comment:
what does `past` mean here? Is there a way to describe this more accurately? Or at least the `help` text should reflect this. Default is set to `past` and the help text mentions `current time`.

Reply:
"past" means the time in time stamper in Header is the time when recording happened. "current time" means the time in time stamper in Header is current system time. Some packages like robot_localization deals with current system time, therefore we want to replace the time stamper as current time so that the rosbag can be directly played and as data input in those packages. @Karsten1987